frontend/src/pages/Domains/domainList.ts

```diff
@@ -0,0 +1,37 @@
+export const domainList = [
```

Comment:
Add a comment on top to make it clear that this list contains dummy data and we need to update it. Preferably create an issue and link it here:
```
// todo(link/to/the/issue): update the list to contain correct data.
```